HashRegistrarSimplified.sol

```diff
@@ -305,11 +302,9 @@ contract Registrar {
         if (address(sealedBids[msg.sender][sealedBid]) > 0 ) throw;
         if (msg.value < minPrice) throw;
         // creates a new hash contract with the owner
-        Deed newBid = new Deed(msg.value);
+        Deed newBid = (new Deed).value(msg.value)();
```

Comment:
It lets you send value along with a creation call. Turns out it's been in since at least Solidity 0.4.0!